<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lcQMinifigDialog</class>
 <widget class="QDialog" name="lcQMinifigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minifig Wizard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="hatsType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="lcQColorPicker" name="hatsColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="hatsAngle">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="9">
       <widget class="QFrame" name="minifigFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QDoubleSpinBox" name="headAngle">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="lcQColorPicker" name="headColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QComboBox" name="headType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="hats2Type">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="lcQColorPicker" name="hats2Color">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="hats2Angle">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="neckType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="lcQColorPicker" name="neckColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="lcQColorPicker" name="bodyColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QComboBox" name="bodyType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="larmType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="lcQColorPicker" name="larmColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="larmAngle">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QDoubleSpinBox" name="rarmAngle">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="lcQColorPicker" name="rarmColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QComboBox" name="rarmType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QComboBox" name="lhandType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="lcQColorPicker" name="lhandColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QDoubleSpinBox" name="lhandAngle">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QDoubleSpinBox" name="rhandAngle">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="lcQColorPicker" name="rhandColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QComboBox" name="rhandType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QComboBox" name="lhandaType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="lcQColorPicker" name="lhandaColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QDoubleSpinBox" name="lhandaAngle">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QDoubleSpinBox" name="rhandaAngle">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="lcQColorPicker" name="rhandaColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="QComboBox" name="rhandaType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QComboBox" name="body2Type">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="lcQColorPicker" name="body2Color">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="lcQColorPicker" name="body3Color">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="6">
       <widget class="QComboBox" name="body3Type">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QComboBox" name="llegType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="lcQColorPicker" name="llegColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QDoubleSpinBox" name="llegAngle">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QDoubleSpinBox" name="rlegAngle">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="lcQColorPicker" name="rlegColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="6">
       <widget class="QComboBox" name="rlegType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QComboBox" name="llegaType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="lcQColorPicker" name="llegaColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QDoubleSpinBox" name="llegaAngle">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QDoubleSpinBox" name="rlegaAngle">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="5">
       <widget class="lcQColorPicker" name="rlegaColor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="6">
       <widget class="QComboBox" name="rlegaType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="TemplateGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Templates</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="TemplateComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TemplateSaveButton">
        <property name="text">
         <string>Save...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TemplateDeleteButton">
        <property name="text">
         <string>Delete...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TemplateImportButton">
        <property name="text">
         <string>Import...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TemplateExportButton">
        <property name="text">
         <string>Export...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lcQColorPicker</class>
   <extends>QToolButton</extends>
   <header>lc_qcolorpicker.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>hatsType</tabstop>
  <tabstop>hatsColor</tabstop>
  <tabstop>hatsAngle</tabstop>
  <tabstop>headAngle</tabstop>
  <tabstop>headColor</tabstop>
  <tabstop>headType</tabstop>
  <tabstop>hats2Type</tabstop>
  <tabstop>hats2Color</tabstop>
  <tabstop>hats2Angle</tabstop>
  <tabstop>neckType</tabstop>
  <tabstop>neckColor</tabstop>
  <tabstop>bodyColor</tabstop>
  <tabstop>bodyType</tabstop>
  <tabstop>larmType</tabstop>
  <tabstop>larmColor</tabstop>
  <tabstop>larmAngle</tabstop>
  <tabstop>rarmAngle</tabstop>
  <tabstop>rarmColor</tabstop>
  <tabstop>rarmType</tabstop>
  <tabstop>lhandType</tabstop>
  <tabstop>lhandColor</tabstop>
  <tabstop>lhandAngle</tabstop>
  <tabstop>rhandAngle</tabstop>
  <tabstop>rhandColor</tabstop>
  <tabstop>rhandType</tabstop>
  <tabstop>lhandaType</tabstop>
  <tabstop>lhandaColor</tabstop>
  <tabstop>lhandaAngle</tabstop>
  <tabstop>rhandaAngle</tabstop>
  <tabstop>rhandaColor</tabstop>
  <tabstop>rhandaType</tabstop>
  <tabstop>body2Type</tabstop>
  <tabstop>body2Color</tabstop>
  <tabstop>body3Color</tabstop>
  <tabstop>body3Type</tabstop>
  <tabstop>llegType</tabstop>
  <tabstop>llegColor</tabstop>
  <tabstop>llegAngle</tabstop>
  <tabstop>rlegAngle</tabstop>
  <tabstop>rlegColor</tabstop>
  <tabstop>rlegType</tabstop>
  <tabstop>llegaType</tabstop>
  <tabstop>llegaColor</tabstop>
  <tabstop>llegaAngle</tabstop>
  <tabstop>rlegaAngle</tabstop>
  <tabstop>rlegaColor</tabstop>
  <tabstop>rlegaType</tabstop>
  <tabstop>TemplateComboBox</tabstop>
  <tabstop>TemplateSaveButton</tabstop>
  <tabstop>TemplateDeleteButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>lcQMinifigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>lcQMinifigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
